packages/fxa-settings/src/setupTests.tsx

```diff
@@ -3,3 +3,12 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 import '@testing-library/jest-dom/extend-expect';
+import { FtlMsgProps } from 'fxa-react/lib/utils';
+
+jest.mock('fxa-react/lib/utils', () => ({
```

Comment:
I was going to wait until we'd used this in a lot more components before mocking this preemptively for every single test, but, I ran the tests 4x times with the mocks in the individual files where I need them and got a runtime ranging 7.3s - 8.2s and then ran it 4x times with this change and got a runtime ranging 7.4s - 7.8s, so the difference seems negligible.